src/zino/statemodels.py

```diff
@@ -29,13 +29,31 @@ class InterfaceState(StrEnum):
     LOWER_LAYER_DOWN = "lowerLayerDown"
 
 
+class BFDSessState(Enum):
+    """The set of allowable BFD session states"""
+
+    ADMINDOWN = "adminDown"
+    DOWN = "down"
+    INIT = "init"
+    UP = "up"
+    NOSESSION = "noSession"
```

Comment:
"noSession" has never been seen in the wild. Should it be added to zinolib?

Reply:
yes, no session is new just to make it clear that we know that there is no bfd session for that port. Session rows can completely disappear from the MIB, so its valid for a state to go from UP to not existing. The alternative is to just have the session state field be None, but I think its clearer this way.

 Im not fully caught up on zinolib but it might be relevant to add it there